gno.land/pkg/integration/testdata/assertorigincall.txtar

```diff
@@ -3,7 +3,7 @@
 #
 # | Num | Msg Type |      Call from      | Entry Point            | Result |
 # |-----|:--------:|:-------------------:|:----------------------:|:------:|
-# | 1   | MsgCall  | wallet direct       | myrealm.A()            | PANIC  |
+# | 1   | MsgCall  | wallet direct       | myrealm.A()            | pass   |
```

Comment:
Should panic - this file should stay the same except for gas changes.

Reply:
I'm confused about this(it doesn't make sense to me). I've reverted my change to `AssertOriginCall`: https://github.com/gnolang/gno/commit/5d1c3e8db79c5ada5b35578597952802d4d75c7e